riscv/csrs.cc

```diff
@@ -1317,3 +1317,16 @@ void sstateen_csr_t::verify_permissions(insn_t insn, bool write) const {
 
   hstateen_csr_t::verify_permissions(insn, write);
 }
+
+// implement class fcsr_csr_t
+fcsr_csr_t::fcsr_csr_t(processor_t* const proc, const reg_t addr, csr_t_p upper_csr, csr_t_p lower_csr, const unsigned upper_lsb):
+  composite_csr_t(proc, addr, upper_csr, lower_csr, upper_lsb) {
+}
+
+
```

Comment:
An earlier commit removed multiple blank lines, and this adds some back!

Reply:
Sorry for this mistake. I'll fix it.